src/read/archive.rs

```diff
@@ -105,6 +196,12 @@ impl<'data, R: ReadRef<'data>> ArchiveFile<'data, R> {
             } else {
                 // TODO: This could still be a BSD file. We leave this as unknown for now.
             }
+        } else if tail < len && file.kind == ArchiveKind::Xcoff {
+            // Read and consume fixed size header.
+            let _fixed_size_header = data
+                .read::<archive::BigFixedLengthHeader>(&mut tail)
+                .read_error("Invalid fixed sized archive header")?;
```

Comment:
Why isn't this being used? In particular, why isn't `BigFixedLengthHeader::fstmoff` used to locate the first member?

Reply:
> It is not needed. First member is always 128 du to Fixed-Length Header size (see documentation). Next member uses absolute offset. When I have written the patch, data was consumed sequentially. With sequential read, we can use size, and it is more consistant with other archive kind. But it is also possible to use next offset as we now know the absolute offset.

Are you sure it isn't needed? What happens when archive members are deleted? My understanding of that documentation is that when members are deleted, it is possible for their header to remain in the file and be added to the free linked list instead. This would mean that using sequential read is wrong for that case, because it will try to read the deleted member.